selectel/resource_selectel_domains_domain_v1.go

```diff
@@ -0,0 +1,106 @@
+package selectel
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	v1 "github.com/selectel/domains-go/pkg/v1"
+	"github.com/selectel/domains-go/pkg/v1/domain"
+)
+
+const domainsEndpoint = "https://api.selectel.ru/domains/v1"
+
+func resourceDomainsDomainV1() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDomainsDomainV1Create,
+		Read:   resourceDomainsDomainV1Read,
+		Update: resourceDomainsDomainV1Update,
+		Delete: resourceDomainsDomainV1Delete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceDomainsDomainV1Create(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	ctx := context.Background()
+	client := v1.NewDomainsClientV1(config.Token, domainsEndpoint)
```

Comment:
It is better to provide a wrapper over this function in the same way as it's done for the Resell API client:
https://github.com/terraform-providers/terraform-provider-selectel/blob/master/selectel/config.go#L36